### PR TITLE
recipes-core: containers-tags: update tags for scarthgap

### PR DESCRIPTION
**Pull request by charles2910**
Summary of changes:

- We are not appending anything to container tags anymore;
- We don't need vivante specific tags anymore because every platform shares the same major;
- We don't use portainer anymore;
- TCB is not supposed to be run from within Torizon so it's unneeded;
- Remove duplicated variables;
- dotnet-uno was replaced by dotnet-wayland;
- We have new GTK 3 containers;
- We have only one dotnet8-debug image now.

Related-to: TCCP-840, TCCP-841, TCCP-706

### CODE REVIEW

**Review by EdTheBearded — DISMISSED** [review]

**Comment by EdTheBearded** [comment]
Nice cleanup in the file!
@charles2910, could you also, please, cherry-pick this PR to our master branch?

**Review by jsrc27 — DISMISSED** [review]
LGTM

**Comment by charles2910** [comment]
> Nice cleanup in the file! @charles2910, could you also, please, cherry-pick this PR to our master branch?

There we go: https://github.com/torizon/meta-toradex-torizon/pull/158

**Comment by leonheldattoradex** [comment]
Hey y'all, I've drafted this because this is not released yet. Not that it makes a big difference, but I don't want to redo commits because we missed something or our side, so please wait until we get our stuff together for the release. Thanks!

**Review by rborn-tx — DISMISSED** [review]
LGTM

File: recipes-core/torizon-conf/torizon-conf/containers-tags.sh [hunk 1]
```diff
@@ -1,38 +1,21 @@
...
+export CT_TAG_DEBIAN="4"
+export CT_TAG_DOTNET_8="4"
+export CT_TAG_DOTNET_8_DEBUG="4"
+export CT_TAG_DOTNET_8_GTK_3="4"
+export CT_TAG_DOTNET_8_WAYLAND="4"
+export CT_TAG_GRAPHICS_TESTS="4"
+export CT_TAG_QT5_WAYLAND="4"
+export CT_TAG_QT5_WAYLAND_EXAMPLES="4"
+export CT_TAG_QT6_WAYLAND="4"
+export CT_TAG_QT6_WAYLAND_EXAMPLES="4"
```
Comment (charles2910) [inline recipes-core/torizon-conf/torizon-conf/containers-tags.sh:15]: @leonheldattoradex remove for now?

Comment (leonheldattoradex) [reply recipes-core/torizon-conf/torizon-conf/containers-tags.sh:15]: It's fine to leave it in, we'll release shortly after.

**Review by EdTheBearded — APPROVED** [review]

**Review by leonheldattoradex — APPROVED** [review]

**Pull request merged**